### DISCUSSION TITLE
C# | Using Entity Framework with PostgreSQL Database

### DISCUSSION THREAD

**Discussion by hbolajraf** [hbolajraf/hbolajraf.github.io#34] [Announcements]
Hello

**Comment by Alteri678** [comment]
Umbraco CMS 15.2 +Debian 12 +PostgreSQL.   How to configure?

**Reply by Alteri678** [reply]
appsettings.json ：
"ConnectionStrings": {
  "umbracoDbDSN": "Server=localhost;Port=5432;Database=umbraco_db;User Id=cirri;Password=Fax60590029;Include Error Detail=true;"
},
"Umbraco": {
  "CMS": {
    "Global": {
      "DatabaseFactoryServerVersion": "17.0",
      "DatabaseFactoryProvider": "Npgsql.EntityFrameworkCore.PostgreSQL"
    }
  }
}




Program.cs



？？？
